packages/pluggableWidgets/popup-menu-native/src/__tests__/PopupMenu.spec.tsx

```diff
@@ -60,25 +72,30 @@ describe("Popup menu", () => {
                 }
             ];
             const component = render(<PopupMenu {...defaultProps} style={customStyle} />);
-
-            expect(component.toJSON()).toMatchSnapshot();
+            expect(component).toMatchSnapshot();
         });
     });
 
-    describe("with custom items", () => {
+    describe("in custom mode", () => {
         beforeEach(() => {
             defaultProps.renderMode = "custom";
         });
 
-        it("renders", () => {
+        it("renders only custom items", () => {
             const component = render(<PopupMenu {...defaultProps} />);
 
-            expect(component.toJSON()).toMatchSnapshot();
+            const basicItems = component.queryAllByTestId(basicItemTestId);
+            const dividers = component.UNSAFE_queryAllByType(MenuDivider);
```

Comment:
I read up on it. Unsafe is called that way because it means you hace knowledge about the internal workings of the app. It is against the visionn of the library owners. So, not advised, but still works.